libraries/botframework-connector/src/auth/jwtTokenProviderFactory.ts

```diff
@@ -8,6 +8,7 @@
 
 import { DefaultAzureCredential } from '@azure/identity';
 import { JwtTokenProviderFactoryInterface } from './jwtTokenProviderFactoryInterface';
+import * as assert from 'assert';

```

Comment:
`import {ok} from 'assert'` and then call `ok()` which is the same as `import * as assert from 'assert'; assert(...)`